production/loki-mixin/dashboards/loki-reads-resources.libsonnet

```diff
@@ -2,14 +2,21 @@ local grafana = import 'grafonnet/grafana.libsonnet';
 local utils = import 'mixin-utils/utils.libsonnet';
 
 (import 'dashboard-utils.libsonnet') {
+  local index_gateway_pod_matcher = if !$._config.ssd then 'container="index-gateway"' else 'container="loki", pod=~"(enterprise-logs|loki)-read.*"',
```

Comment:
WDYT about flipping these so we check against `$._config.ssd` instead of it's inverse? I always find checking the inverse to take just a bit more mental effort, so in a case where there's an action for each branch of the conditional I'd prefer not asserting against the inverse.

Reply:
I don't have a preference here. I took this approach so the microservices matcher was before the SSD matcher since: 
- We run all our prod and most of our ops/dev cells in microservices mode
- The matcher for microservices is shorter and easier to read/understand

I'll flip them

Reply:
I just left a couple of exceptions:

- hiddenRows:
```jsonnet
hiddenRows:: [
  'Cassandra',
] + if !$._config.ssd then [] else [
  'Ingester',
],
```

- addRowIf
```jsonnet
.addRowIf(
  !$._config.ssd,
```

Reply:
yeah, I agree with the ones you left. thanks!